mac/Focalboard/ViewController.swift

```diff
@@ -107,8 +107,14 @@ class ViewController:
 
 	private func updateSessionTokenAndUserSettings() {
 		let appDelegate = NSApplication.shared.delegate as! AppDelegate
+		let url = URL(string: "http://localhost:\(appDelegate.serverPort)/")!
```

Comment:
This should currently not be a practical problem but I think you don't actually have to create a `URL` object since the only place you're using the result is in the `"...\(url)..."` string interpolation below. I believe what happens in the interpolation is it converts the `URL` object to its `description` property (which happens to be the URL as a `String`). To prevent that implicit conversion, you could just use `let url = "http://localhost:\(appDelegate.serverPort)/"`.